README.md

```diff
@@ -43,14 +43,26 @@ development on this starter kit.
 
 ### Local starter kit development
 
+- Before using please understand:
+  - Starter Kit can be thought of as a "utility" that is installed on your computer that helps you to create new website projects.
+  - Once installed, when you run Starter Kit it will scaffold a basic app that adheres to the goals in the #Objectives and #Features sections above.
```

Comment:
```suggestion
  - Once installed, when you run Starter Kit it will scaffold a basic app that adheres to the goals in the [Objectives](#objectives) and [Features](#features) sections above.
```